src/xenia/gpu/command_processor.cc

```diff
@@ -1179,9 +1179,11 @@ bool CommandProcessor::ExecutePacketType3_DRAW_INDX(RingBuffer* reader,
   // initiate fetch of index buffer and draw
   // if dword0 != 0, this is a conditional draw based on viz query.
   // This ID matches the one issued in PM4_VIZ_QUERY
-  // ID = dword0 & 0x3F;
-  // use = dword0 & 0x40;
   uint32_t dword0 = reader->ReadAndSwap<uint32_t>();  // viz query info
+  uint32_t viz_id = dword0 & 0x3F;
```

Comment:
These will be unreferenced local variables and trigger warnings on some compilers.